src/resolve/resolved-dns-packet.c

```diff
@@ -730,8 +730,9 @@ int dns_packet_append_opt(
                 uint16_t max_udp_size,
                 bool edns0_do,
                 bool include_rfc6975,
+                const char *nsid,
```

Comment:
Doesn't seem to work for some reason:
```console
$ dig +nsid localhost @127.0.0.53

; <<>> DiG 9.11.24-RedHat-9.11.24-2.fc34 <<>> +nsid localhost @127.0.0.53
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 9661
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;localhost.			IN	A

;; ANSWER SECTION:
localhost.		0	IN	A	127.0.0.1

;; Query time: 2 msec
;; SERVER: 127.0.0.53#53(127.0.0.53)
;; WHEN: Fri Nov 27 18:38:17 CET 2020
;; MSG SIZE  rcvd: 54
```

Reply:
Hmm, I just checked again here. Seems to work fine. I am puzzled. I am sure you ran the correct version of systemd-resolved when you tested this?

Reply:
I'll recheck.